Pipfile

```diff
@@ -0,0 +1,29 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+
+[dev-packages]
+black = "*"
+pylint = "*"
+flake8 = "*"
+pydocstyle = "*"
+pytest = "*"
+PyGithub= "*"
+streamlit = "*"
+numpy = "*"
+pandas = "*"
+fastapi = "*"
+uvicorn = "*"
+
+[requires]
+python_version = "3.7"
+
+[pipenv]
+allow_prereleases = true
+
+[scripts]
+lint = "./scripts/lint.sh"
+test = "./scripts/test.sh"
```

Comment:
There are no these two files in the master branch currently

Reply:
I have removed these files from `Pipfile`. They can be added back if/when we add these files to the master branch.